testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/toolbarIntegration.tests.js

```diff
@@ -108,6 +108,27 @@ export default function() {
                 .trigger('dxclick');
         });
 
+        test('No active buttons when cursor placed on right side of formatted text(T1157840)', function(assert) {
+            const initialSelection = {
```

Comment:
I don't see y it should be "initial"
```suggestion
            const selection = {
```

Reply:
let's rename to **firstSeection**? I want to show what on second changing we change selection on index + length of _initial_ selection (prev selection)

Reply:
ok, nevermind